README.md

```diff
@@ -48,6 +59,31 @@ Images are composed of multiple layers which are stored as individual files on t
 
 In its core Spegel is a pull only OCI registry which runs locally on every Node in the Kubernetes cluster. Containerd is configured to use the local registry as a mirror, which would serve the image from within the cluster or from the source registry.
 
+## Alternatives
+
+### Private Registry 
+
+A common practice, especially for larger enterprises, is to run a private registry like Harbor to replica images from public registries. Storing them within the private network close to the cluster.
+This is a great option for those who have the time and budget to invest in running and managing the infrastructure. For others, it may be a good practice but unattainable in reality.
+Spegel does not aim to replace projects like [Harbor](https://github.com/goharbor/harbor) or [Zot](https://github.com/project-zot/zot) but instead complements them. Having a persistent copy of public images stored geographically close to a cluster is great. Spegel will however enable
+nodes to pull from images closer as long as the images are somewhere within the cluster. Additionally, there is no guarantee that a self-managed private registry is never unavailable. In these scenarios
+running Spegel is like having both belt and braces.
+
+### Dragonfly
+
+[Dragonfly](https://github.com/dragonflyoss/Dragonfly2) is a great project that has been around for a while. In some aspects, Spegel takes inspiration from the work done by Dragonfly. 
+The difference is that Spegel aims to solve a smaller problem set. While it may mean fewer features it also means fewer moving components. Dragonfly requires both Redis and Mysql which 
+increases the resource consumption and burden on end users to manage additional resources. It also increases the risk of errors occurring during critical moments. The benefit of Spegel
+is that it is stateless meaning that any temporary failure of nodes and communication should be easily resolved automatically.
+
+### Kraken
+
+[Kraken](https://github.com/uber/kraken) implements a similar solution to Spegel with its P2P agent component. It is however not heavily maintained, meaning that new features and security updates will not be added.
```

Comment:
Not sure how "heavily" should be interpreted? It appears to be maintained, in that it had a commit in January, but no release since 2020. It certainly appears to be "maintained", but maybe not "actively" maintained. Anyway, I don't have a better suggestion than "heavily", I'm just not sure how a reader should understand that choice of word.

Reply:
I don't know myself, as there seems to be some activity but no real community engagement looking at the issues. I will leave this for now until we come up with something better.